Formula/python-gdbm@3.11.rb

```diff
@@ -0,0 +1,53 @@
+class PythonGdbmAT311 < Formula
+  desc "Python interface to gdbm"
+  homepage "https://www.python.org/"
+  url "https://www.python.org/ftp/python/3.11.0/Python-3.11.0.tgz"
+  sha256 "64424e96e2457abbac899b90f9530985b51eef2905951febd935f0e73414caeb"
+  license "Python-2.0"
+
+  livecheck do
+    formula "python@3.11"
+  end
```

Comment:
Not seen this DSL before but: I really like it. To attempt a nerd-snipe: if anyone else agrees with me it might be cool to allow a similar DSL for `install` or `post_install` or similar for versioned formulae to share more code. If anyone else agrees: I'll write up an issue. CC @Homebrew/maintainers for thoughts.

Reply:
It's [documented](https://docs.brew.sh/Brew-Livecheck#referenced-formulacask) but rarely used for versioned formulae; it's instead found in those that are derived from / entangled with another formula, e.g. [lemon](https://formulae.brew.sh/formula/lemon).

Reply:
@EricFromCanada Yeh, I was more thinking a differing DSL something like

```ruby
def install
  formula "python@3.11"
end
```

or something